src/App.js

```diff
@@ -29,23 +29,23 @@ function App() {
       <Grid item xs={12} align="center" style={{marginTop:'3vh'}}>
         <Button variant="outlined" color="primary" disabled={true} style={{marginInlineEnd:'10px'}}> Uniswap </Button>
         <Button variant="outlined" color="primary" disabled={false} href="https://t.me/ZZZfinance" style={{marginInlineEnd:'10px'}}> Telegram </Button>
-        <Button variant="outlined" color="primary" disabled={false} href="https://gov.zzz.finance"> Governance </Button>
+        <Button variant="outlined" color="primary" disabled={false} href="http://pool.zzz.finance"> Pools </Button>
```

Comment:
can you change this to https:// link instead of http://

Reply:
oh ! sure